source/book/include/marlin/book/BookStore.h

```diff
@@ -126,20 +130,33 @@ namespace marlin {
        *  @brief get Entry from key.
        *  @throw BookStoreException key not exist in Store.
        */
-      details::Entry &get( const EntryKey &key ) { return get( key.hash ); }
+      details::Entry &get( const EntryKey &key ) const { return get( key.idx ); }
 
       /**
        *  @brief get Entry from key.
        *  @throw BookStoreException key not exist in Store.
        */
-      details::Entry &get( std::size_t const key ) {
+      details::Entry &get( std::size_t const idx ) const {
         try {
-          return *_entries[key] ;
+          return *_entries[idx] ;
         } catch ( const std::out_of_range & ) {
           MARLIN_BOOK_THROW( "Invalid key." ) ;
         }
       }
 
+      /**
+       *  @brief 
+       *  @throw BookStoreException key not exist in Sore
+       */
+      const std::shared_ptr<details::Entry> 
+        &getPtr( std::size_t const idx ) const {
+        try {
+          return _entries[idx] ;
+        } catch ( const std::out_of_range& ) {
+          MARLIN_BOOK_THROW( "Invalid key." ) ;
+        }
+      }
+
```

Comment:
make EntryPointer accessible for cleaner usage in `Handle<Entry<T>>` construction

Reply:
Documentation string ?

Reply:
What do you mean by this?
the throw is documented with \@throw above for Doxygen ...